cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -87,6 +89,10 @@ func GetResourceTypeDisplayName(resourceType AzureResourceType) string {
 		return "AKS Managed Cluster"
 	case AzureResourceTypeAgentPool:
 		return "AKS Agent Pool"
+	case AzureResourceTypeCognitiveServiceAccount:
+		return "Cognitive Search"
+	case AzureResourceTypeSearchService:
+		return "Search Service"
```

Comment:
We were previously using the form of `Azure Service Name` and recently changed to only `Service Name` (removing the `Azure` word).

This is not such a big deal, but it is a tiny detail we might want to define from @savannahostrowski / @ellismg .

Not a blocker for this PR, but we should set the pattern for GA